compiler/src/dotty/tools/dotc/Compiler.scala

```diff
@@ -37,7 +37,6 @@ class Compiler {
   /** Phases dealing with the frontend up to trees ready for TASTY pickling */
   protected def frontendPhases: List[List[Phase]] =
     List(new FrontEnd) ::           // Compiler frontend: scanner, parser, namer, typer
-    List(new YCheckPositions) ::    // YCheck positions
```

Comment:
Why did we have to remove the phase? Does that mean positions are not checked anymore?

Reply:
YCheckPositions used to check that the sources we have on the trees align with the source that is in the `call` of the `Inlined` trees. Initially, this was used when we added sources to the trees to make sure those where correct and catch `Inlined` trees that had an empty/non-nonempty `call` when they should not. This was important to make sure the positions of the trees where properly unpickled and that the transformation phases did not lose information about the sources on the `Inlined.call`s before erasure.
This invariant does not hold after `PostTyper` anymore as all calls are empty and all the inlined source logic relies on the sources of the trees instead of the calls. Therefore we can't check what we were originally checking and we don't need to as we use exclusively the sources of the trees.